packages/dialect-react-ui/package.json

```diff
@@ -1,13 +1,23 @@
 {
   "name": "@dialectlabs/react-ui",
-  "version": "0.1.15",
+  "version": "0.2.0",
   "description": "Dialect's react UI components",
   "license": "MIT",
   "private": false,
-  "main": "./lib/index.js",
-  "types": "./lib/index.d.ts",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/es/index.js",
+  "types": "./lib/es/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/es/index.js",
+      "require": "./lib/cjs/index.js"
+    },
+    "./styles": "./lib/index.css"
```

Comment:
I lean toward ```"./styles.css": "./lib/index.css"``` to be extreamly clear what user imports and to avoid issues with import base on file extention

Reply:
Unfortunately, I think we can't specify the `.css` part, but I can double-check that.

I agree that it's great to be explicit, however I think that styles.css might also be confusing, since it's also a misconception in terms of the actual file.

Reply:
@fsher Solana use the ".css" for example https://github.com/solana-labs/wallet-adapter/blob/e15287221b90f8d7bee60726d251a5b4c76d243a/packages/ui/react-ui/package.json#L17

Reply:
@Kiryous Thank you so much! Perfect, I'll update that now.

Reply:
Updated to ./index.css for better file representation